splitJoinSliceSplice/challenges-05.test.js

```diff
@@ -0,0 +1,301 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named howMuchPencil that takes in a string, as written on the side of a pencil.
+// As you sharpen the pencil, the string will become shorter and shorter, starting by removing the
+// first letter.
+//
+// Your function should use slice within a loop and return an array of each successive string
+// result from losing letters to the sharpener, until nothing is left.
+//
+// For example, if the input is 'Welcome', the output will be:
+// ['Welcome', 'elcome', 'lcome', 'come', 'ome', 'me', 'e', ''].
+// ------------------------------------------------------------------------------------------------
+
+const howMuchPencil = (name) => {
+  let answer = [];
+  for (let i = 0; i < name.length + 1; i++) {
+    answer.push(name.slice(i));
+  }
+  return answer;
+};
+/// not sure reading the let answer
+// was spelling answer wrong
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function name wordsToCharList that, given a string as input, returns a new array where every element is a
+// character of the input string.
+//
+// For example, wordsToCharList('gregor') returns ['g','r','e','g','o','r'].
+// ------------------------------------------------------------------------------------------------
+
+const wordsToCharList = (input) => {
+  return input.split('');
+
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 3
+//
+// Write a function named totalSumCSV that, given a string of comma-separated values (CSV) as input
+// (e.g. "1,2,3"), returns the total sum of the numeric values (e.g. 6).
+// ------------------------------------------------------------------------------------------------
+
+const totalSumCSV = (input) => {
+  let total = 0; {

```

Comment:
Not sure what these extra curly braces are doing here! I don't think you want these.